ui/imports/shared/stores/TransactionStore.qml

```diff
@@ -36,6 +36,8 @@ QtObject {
     property var disabledChainIdsFromList: []
     property var disabledChainIdsToList: []
 
+    property var assets: walletSectionAssets.assets
```

Comment:
Wow, not an easy one to spot.

Reply:
@Khushboo-dev-cpp I don't see any usage of `assets`?